loc/tests/test_schema.py

```diff
@@ -332,3 +332,37 @@ def test_email_letter_works(db, graphql_client, mailoutbox):
     )['data']['emailLetter']
     assert result == {'errors': [], 'recipients': ['boop@jones.com']}
     assert len(mailoutbox) == 1
+
+
+def _exec_relief_attempts_form(graphql_client, input):
+    return graphql_client.execute(
+        """
+        mutation MyMutation($input: ReliefAttemptsInput!) {
+            output: reliefAttempts(input: $input) {
+                errors { field, messages }
+                session {
+                    onboardingInfo {
+                        hasCalled311
+                    }
+                }
```

Comment:
I guess technically you can get rid of this whole part since you don't actually reference it in your tests, I think?  Up to you tho.